src/recovery.rs

```diff
@@ -307,13 +314,33 @@ impl Recovery {
         self.loss_detection_timer
     }
 
-    pub fn cwnd(&self) -> usize {
+    pub fn cwnd_available(&self) -> usize {
         // Ignore cwnd when sending probe packets.
         if self.loss_probes.iter().any(|&x| x > 0) {
             return std::usize::MAX;
         }
 
-        self.cwnd.saturating_sub(self.bytes_in_flight)
+        self.cwnd().saturating_sub(self.bytes_in_flight())
+    }
+
+    // Shortcut to CC module's cwnd()
+    pub fn cwnd(&self) -> usize {
```

Comment:
All of these "shortcuts" just make it harder to read the code IMO, as it's never clear where a given value comes from without having to follow the shortcuts every time.

Reply:
By removing the `Option` as per a previous comment, this wouldn't be needed as we could just do `self.cc.method()` directly btw.

Reply:
All shortcut added to access CC modules is removed. Now simply use
recovery.cwnd -> recovery.cc.cwnd() and so on.